src/api.jl

```diff
@@ -161,13 +182,14 @@ function infer(fieldids::Vector{Tuple{Int, Int, Int}},
 
     # If there are no objects of interest, return early.
     if length(target_sources) == 0
-        return Dict{Int, Dict}()
+        return Dict{Int, Dict}(), (0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0)
```

Comment:
It's fine to hold off on writing unit tests, but I do think it's going to be confusing if `celeste.jl` contains code that isn't just about parsing arguments and calling the right function in `api.jl`.